TV/GameModes/BriansGames/BodyBuilder/GameStates/Gameplay_GS.cs

```diff
@@ -0,0 +1,231 @@
+                        {
+                            Prompt = "Which body part do you want to trade?",              
+                            Answers = new string[]
+                            {
+                                HtmlImageWrapper(PlayerTrade.BodyPartDrawings[DrawingType.Head].Drawing,headWidth,headHeight),
+                                HtmlImageWrapper(PlayerTrade.BodyPartDrawings[DrawingType.Body].Drawing,bodyWidth,bodyHeight),
+                                HtmlImageWrapper(PlayerTrade.BodyPartDrawings[DrawingType.Legs].Drawing,legWidth,legHeight),
+                                "None"
+
+                            },
+                        }
+
+                    },
+                    SubmitButton = true,
+                };
+            };
+        }
+
+        private string HtmlImageWrapper(string image, int width = 240, int height = 240)
+        {
+            return Invariant($"<img width=\"{width}\" height=\"{height}\" src=\"{image}\"/>");
+        }
+
+        private List<Gameplay_Person> UnassignedPeople { get; set; } = new List<Gameplay_Person>();
+        Dictionary<User, Gameplay_Person> AssignedPeople { get; set; } = new Dictionary<User, Gameplay_Person>();
+        Dictionary<User, int> UsersToSeatNumber { get; set; } = new Dictionary<User, int>();
+        public Gameplay_GS(Lobby lobby, List<Setup_Person> setup_PeopleList, Action<User, UserStateResult, UserFormSubmission> outlet = null) : base(lobby, outlet)
+        {
+            this.AssignPeople(setup_PeopleList);
+            this.AssignSeats();
+            
+            this.Entrance = AddGameplayCycle();
+
+            /*UserStateTransition waitForUsers = new WaitForAllPlayers(lobby: this.Lobby, outlet: this.Outlet);
+            waitForUsers.AddStateEndingListener(() => this.UpdateScores());
+            pickDrawing.SetOutlet(waitForUsers.Inlet);
+            waitForUsers.SetOutlet(this.Outlet);
+
+            var unityImages = new List<UnityImage>();
+            foreach ((string id, ConcurrentDictionary<string, string> colorMap) in this.SubChallenge.TeamIdToDrawingMapping)
+            {
+                unityImages.Add(new UnityImage
+                {
+                    Base64Pngs = new StaticAccessor<IReadOnlyList<string>> { Value = this.SubChallenge.Colors.Select(color => colorMap[color]).ToList() },
+                    ImageIdentifier = new StaticAccessor<string> { Value = id.ToString() },
+                    BackgroundColor = new StaticAccessor<IReadOnlyList<int>> { Value = new List<int> { 255, 255, 255 } }
+                });
+            }
+            this.UnityView = new UnityView
+            {
+                ScreenId = new StaticAccessor<TVScreenId> { Value = TVScreenId.ShowDrawings },
+                UnityImages = new StaticAccessor<IReadOnlyList<UnityImage>> { Value = unityImages },
+                Title = new StaticAccessor<string> { Value = "Behold!" },
+            };*/
```

Comment:
Not sure if you want to tackle the Unity view in this PR or a separate one.

```suggestion
            // TODO: Add unity view
            /*
            var unityImages = new List<UnityImage>();
            foreach ((string id, ConcurrentDictionary<string, string> colorMap) in this.SubChallenge.TeamIdToDrawingMapping)
            {
                unityImages.Add(new UnityImage
                {
                    Base64Pngs = new StaticAccessor<IReadOnlyList<string>> { Value = this.SubChallenge.Colors.Select(color => colorMap[color]).ToList() },
                    ImageIdentifier = new StaticAccessor<string> { Value = id.ToString() },
                    BackgroundColor = new StaticAccessor<IReadOnlyList<int>> { Value = new List<int> { 255, 255, 255 } }
                });
            }
            this.UnityView = new UnityView
            {
                ScreenId = new StaticAccessor<TVScreenId> { Value = TVScreenId.ShowDrawings },
                UnityImages = new StaticAccessor<IReadOnlyList<UnityImage>> { Value = unityImages },
                Title = new StaticAccessor<string> { Value = "Behold!" },
            };*/
```